src/Rules/FunctionDefinitionCheck.php

```diff
@@ -61,6 +62,23 @@ public function __construct(
 		$this->checkThisOnly = $checkThisOnly;
 	}
 
+	/**
+	 * @return string[]
+	 */
+	private function getSupportedTypehints(): array
+	{
+		if (PHP_VERSION_ID < 70200) {
```

Comment:
This may be OK as temporary solution but it's wrong solution in the long term. The PHP version used to run PHPStan should be independent of PHP version used by the analyzed code. For example you may run PHPStan with PHP 7.1 but the analyzed code can still use PHP 5.6.

The correct solution is to add `phpVersion` parameter to config with `PHP_VERSION_ID` as default value.

Reply:
I think this is not necessary to do now. Once we have static reflection, it might be worth doing. But I still want to keep a pool of supported PHP versions as small as possible.